src/imfilter.jl

```diff
@@ -25,4 +25,58 @@ function imfilter_gpu(img, krn)
   real.(result[CartesianIndices(finalsize)]) |> Array
 end
 
-const imfilter_kernel = CUDA.functional() ? imfilter_gpu : imfilter_cpu
+function imfilter_opencl(img, krn)
+  # retrieve basic info
+  N = ndims(img)
+  T = ComplexF64
+
+  # retrieve OpenCL info
+  device, ctx, queue = OpenCL.cl.create_compute_context()
```

Comment:
Can we clean all the prefixes OpenCL in the code? I understand that OpenCL exports the cl variable already.